gdc_ng_models/models/notifications.py

```diff
@@ -18,8 +18,14 @@ class Notification(Base):
     level = Column(String)
     dismissible = Column(Boolean, default=True)
     created = Column(
-        DateTime(timezone=True), nullable=False, server_default=text("now()"),
+        DateTime(timezone=True),
+        nullable=False,
+        server_default=text("now()"),
     )
+    start_date = Column(DateTime(timezone=True), nullable=True)
```

Comment:
I'm late to this party, but I'm curious, why is start_date nullable?

Reply:
https://jira.opensciencedatacloud.org/browse/DEV-485?focusedCommentId=48730&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-48730

basically we didn't want to go back through existing notifications and give them misleading start_dates

Reply:
Makes sense 👍